handsontable/src/plugins/nestedHeaders/nestedHeaders.js

```diff
@@ -15,6 +15,8 @@ import StateManager from './stateManager';
 import GhostTable from './utils/ghostTable';
 
 import './nestedHeaders.css';
+import { normalizeRanges } from '../copyPaste/copyableRanges';
```

Comment:
This import will break the plugin's modularization.

Reply:
Done within https://github.com/handsontable/handsontable/pull/10560/commits/4776ac916370d299a557321e66289a6f5a09a461.